ee/maintained-apps/inputs/homebrew/vnc-viewer.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "VNC Viewer",
+  "name": "RealVNC Viewer",
   "unique_identifier": "com.realvnc.vncviewer",
   "token": "vnc-viewer",
   "installer_format": "dmg",
-  "slug": "vnc-viewer/darwin",
+  "slug": "realvnc-viewer/darwin",
   "default_categories": ["Productivity"]
 } 
```

Comment:
Do we want to change the name of this file to `realvnc-viewer.json`?

Reply:
Yep.

Reply:
Done.